### PR TITLE
[AND-192] Add suggestions to GameGenie

### PR DESCRIPTION
**Pull request by tomascarvalho7**
**What does this PR do?**

Add question suggestions to GameGenie, missing strings

**Database changed?**

  No

**Where should the reviewer start?**

- [ ] SuggestionBox.kt

**How should this be manually tested?**

Open the app 

  Flow on how to test this or QA Tickets related to this use-case: [AND-192](https://aptoide.atlassian.net/browse/AND-192)

**What are the relevant tickets?**

  Tickets related to this pull-request: [AND-192](https://aptoide.atlassian.net/browse/AND-192)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 

**Code Review Checklist**

- [ ] Documentation on public interfaces
- [ ] Database changed? If yes - Migration?
- [ ] Remove comments & unused code & forgotten testing Logs
- [ ] Codestyle
- [ ] New Kotlin code has unit tests
- [ ] New flows in presenters unit tests
- [ ] Mappers/Validators with any kind of logic unit tests
- [ ] Functional tests pass

[AND-192]: https://aptoide.atlassian.net/browse/AND-192?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[AND-192]: https://aptoide.atlassian.net/browse/AND-192?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by apkbuilder-lw** [comment]
Can one of the admins verify this patch?

**Comment by apkbuilder-lw** [comment]
Can one of the admins verify this patch?

File: app-games/src/main/java/com/aptoide/android/aptoidegames/gamegenie/presentation/composables/MessageList.kt [hunk 1]
```diff
@@ -27,19 +28,31 @@ fun MessageList(
     contentPadding = PaddingValues(vertical = 8.dp)
   ) {
     itemsIndexed(messages) { idx, message ->
+    itemsIndexed(items = messages, key = { idx, message -> message.apps }) { idx, message ->
```
Comment (phillipsgoncalves) [inline app-games/src/main/java/com/aptoide/android/aptoidegames/gamegenie/presentation/composables/MessageList.kt:31]: this is wrong it is reapeted. The commits were not well done

File: app-games/src/main/java/com/aptoide/android/aptoidegames/gamegenie/presentation/composables/MessageList.kt [hunk 2]
```diff
@@ -27,19 +28,31 @@ fun MessageList(
...
       if (idx == 0) {
         MessageBubble(
           message = null, isUserMessage = false,
           apps = message.apps.map { app -> app.packageName },
           navigateTo = navigateTo,
           onAllAppsFail = onAllAppsFail
+          onAllAppsFail = onAllAppsFail,
```
Comment (phillipsgoncalves) [inline app-games/src/main/java/com/aptoide/android/aptoidegames/gamegenie/presentation/composables/MessageList.kt:38]: also duplicated

File: app-games/src/main/java/com/aptoide/android/aptoidegames/gamegenie/presentation/composables/MessageList.kt [hunk 3]
```diff
@@ -27,19 +28,31 @@ fun MessageList(
...
+        suggestions.forEach { suggestion ->
+          SuggestionBox(suggestion, onSuggestionClick)
+        }
       } else {
         MessageBubble(
           message = message.gpt, isUserMessage = false,
           apps = message.apps.map { app -> app.packageName },
           navigateTo = navigateTo,
           onAllAppsFail = onAllAppsFail
+          onAllAppsFail = onAllAppsFail,
```
Comment (phillipsgoncalves) [inline app-games/src/main/java/com/aptoide/android/aptoidegames/gamegenie/presentation/composables/MessageList.kt:52]: also duplicated

**Comment by phillipsgoncalves** [comment]
This pull request will be closed, since we are reviewing everything at the same time on :
https://github.com/Aptoide/aptoide-client-v8/pull/2156

This solution was adopted because all branches came out of each other so this way we can check everything in a row without having to solve conflicts or rebase 3 times.

**Pull request closed without merging**